action.yml

```diff
@@ -9,6 +9,10 @@ inputs:
   tag:
     description: 'Tag for swift.org builds. Only specifiy when using official Swift toolchains from swift.org'
     required: false
+  os_arch_suffix:
```

Comment:
Can we name this `build_arch`?

Reply:
I think automatically detecting the arch is easier to use. PTAL. @compnerd

Reply:
Done